addons/event_sale/models/event.py

```diff
@@ -25,6 +29,11 @@ def _default_tickets(self):
         except ValueError:
             return self.env['event.event.ticket']
 
+    @api.one
+    @api.depends('event_ticket_ids.seats_max')
+    def _compute_seats_max(self):
+        self.seats_max = sum(ticket.seats_max for ticket in self.event_ticket_ids)
```

Comment:
@antonylesuisse , i think group by is not good here.because  we need sum of seats_max of all event tickets here.
 can we use sql query here that will be faster ?.